examples/transportation/segment/road/road-alley.yaml

```diff
@@ -0,0 +1,22 @@
+---
+id: overture:transportation:segment:1555
+type: Feature
+geometry:
+  type: LineString
+  coordinates: [[0, 0], [1, 1]]
+properties:
+  # Overture properties
+  theme: transportation
+  type: segment
+  update_time: "2023-02-23T00:03:59-08:00"
+  version: 4
+  subtype: road
+  road:
+    class: alley
+    restrictions:
+      access:
+        - allowed:
+            when:
+              using: [at_destination]
+      speed_limits:
+        - max_speed: [15, "mph"]
```

Comment:
Nit: I would be tempted to drop this to simplify the example.

Reply:
I just checked and we have speed limits as low as 0 (obviously bad data) but many 1 km/h with 2.6K having a numeric value <5. I propose we drop it to 1 to just restrict truly invalid numbers.

Reply:
I agree. I'm OK with filing an issue or just dropping it in this PR, whatever makes most sense.

Reply:
I created a separate issue, we have a variety of speed limit issues that deserve a little more thought and it limits this PR to the original intent.